src/user/ZulipAccountsView.js

```diff
@@ -18,89 +18,28 @@ import {
 
 import ZulipPasswordAuthView from './ZulipPasswordAuthView';
 import ZulipDevAuthView from './ZulipDevAuthView';
-
-const STATUS_BAR_HEIGHT = 20;
-const FIELD_HEIGHT = 44;
-
-const styles = StyleSheet.create({
-  container: {
-    flex: 1,
-    flexDirection: 'column',
-    justifyContent: 'center',
-    marginTop: STATUS_BAR_HEIGHT,
-  },
-  error: {
-    justifyContent: 'flex-start',
-    textAlign: 'center',
-    fontSize: 24,
-    padding: 10,
-  },
-  heading1: {
-    textAlign: 'center',
-    fontSize: 24,
-    padding: 10,
-  },
-  heading2: {
-    textAlign: 'center',
-    fontSize: 18,
-    padding: 10,
-  },
-  user: {
-    backgroundColor: '#8ac',
-  },
-  admin: {
-    backgroundColor: '#f88',
-  },
-  userButton: {
-    flex: 1,
-    alignItems: 'center',
-    padding: 10,
-    height: FIELD_HEIGHT,
-    borderWidth: 1,
-    borderColor: '#999',
-    borderRadius: 5,
-  },
-  fieldInput: {
-    flex: 1,
-    borderWidth: 1,
-    borderColor: '#999',
-    padding: 10,
-  },
-  field: {
-    flexDirection: 'row',
-    justifyContent: 'center',
-    alignItems: 'center',
-    height: FIELD_HEIGHT,
-    marginTop: 10,
-    marginLeft: 20,
-    marginRight: 20,
-  },
-});
-
-const ZulipAddAccountButton = (props) =>
-  <View style={styles.field}>
-    <TouchableHighlight
-      style={[styles.userButton]}
-      onPress={props.onPress}
-    >
-      <Text>Next</Text>
-    </TouchableHighlight>
-  </View>;
-
+import styles from './styles';
+import ZulipError from './ZulipError';
+import ZulipButton from './ZulipButton';
 
 class ZulipAccountsView extends React.Component {
   constructor(props) {
     super(props);
 
-    let defaultRealm = '';
+    let defaultRealm = 'https://zulip.com';
```

Comment:
Rolling it back. Text input on mobile is cumbersome, so a sensible default is important.